sample_size/utils/utils.py

```diff
@@ -0,0 +1,94 @@
+from typing import Dict
+
+from sample_size.sample_size_calculator.sample_size_calculator import PowerAnalysisParameters
+from sample_size.sample_size_calculator.sample_size_calculator import SampleSizeCalculators
+
+DEFAULT_ALPHA = 0.05
+
+
+def is_float(value: str) -> float:
+    try:
+        float(value)
+        return True
+    except ValueError:
+        return False
+
+
+def get_raw_input(text: str) -> str:
+    return input(text)
```

Comment:
if this function is just returning input(), we don't actually need an extra function? can just do text() directly when we need it?

Reply:
I didn't find an easy way to test input() or mock input(). `get_raw_input` is created to avoid calling input() directly. Any suggestions on the test?